aws/resource_aws_launch_template_test.go

```diff
@@ -1658,3 +1753,30 @@ resource "aws_launch_template" "test" {
 }
 `, rName, enabled)
 }
+
+func testAccAWSLaunchTemplateConfig_defaultVersion(rInt int, version int) string {
+	return fmt.Sprintf(`
+resource "aws_launch_template" "test" {
+  name = "test_%d"
+  default_version = %d
+  user_data = "bar"
+
+  tags = {
+    test = "baz"
+  }
+}
+`, rInt, version)
+}
+
+func testAccAWSLaunchTemplateconfig_updateDefaultVersion(rInt int) string {
+	return fmt.Sprintf(`
+resource "aws_launch_template" "test" {
+  name = "test_%d"
+  update_default_version = true
+  user_data = "bar"
```

Comment:
Same here

```suggestion
```

Reply:
similarly handled as above comment to ensure the condition where solely changing the `update_default_version` field does not create a new version (step 3/4 of `TestAccAWSLaunchTemplate_updateDefaultVersion`)